lib/errors.js

```diff
@@ -203,6 +203,27 @@ const codes = {
     TypeError
   ),
 
+  FST_ERR_LOG_INVALID_LOGGER_INSTANCE: createError(
+    'FST_ERR_LOG_INVALID_LOGGER_INSTANCE',
+    'loggerInstance only accepts a logger not a configuration object.',
+    500,
+    TypeError
+  ),
+
+  FST_ERR_LOG_INVALID_LOGGER_CONFIG: createError(
+    'FST_ERR_LOG_INVALID_LOGGER_CONFIG',
+    'logger options only accepts a configuration object.',
```

Comment:
Maybe pino needs a validate function for configuration objects. So maybe should be checked on pino if there is such a function and if not add it and then use it. Then we can check the passed object AND then it is not "logger options only accepts a configuration object" but then we could say "logger option is not a valid logger configuration"